client/app/services/service-details/service-details.html

```diff
@@ -7,8 +7,24 @@
         <li class="active">{{vm.title}}</li>
       </ol>
   </span>
+  <span class="service-details-toolbar-buttons">
```

Comment:
These two icons don't appear to line up (from ss in pr) additionally, do they indicate the active view? From this [mockup](https://github.com/ManageIQ/manageiq-design/blob/master/UX/ui-service/Catalogs/design.md) the active view type is indicated by the breadcrumbs toolbar icon changing to that blueish color when selected.  For references on how this is done, take a look at some of that catalog code we had prior to yanking out the list view.